### PR TITLE
Scroll into view by clicking on `queryLink`

### PR DESCRIPTION
**Pull request by a-chabin**

### CODE REVIEW

File: client/view/QueryLink/queryLink.js [hunk 1]
```diff
@@ -10,6 +12,10 @@ links.forEach(item => {
     } else {
       setFormValues({ query: item.innerText.trim() })
     }
+
     submitForm()
+
+    queryContainer.scrollTo({ top: 0 })
```
Comment (ai) [inline client/view/QueryLink/queryLink.js:18]: `behavior: "smooth"`?

File: client/view/QueryLink/queryLink.js [hunk 2]
```diff
@@ -10,6 +12,10 @@ links.forEach(item => {
...
+    queryContainer.scrollIntoView()
```
Comment (ai) [inline client/view/QueryLink/queryLink.js:19]: Why do we have 2 ways of scrolling?

Comment (a-chabin) [reply client/view/QueryLink/queryLink.js:19]: We have 2 scrollable areas

1. Scroll in `queryContainer` (with rules `min-height 100vh; overflow: auto`). On phone screen our scroll position is at the bottom when we are under `queryContainer`.

2. Scroll to top of `queryConainer` in viewport

Did you get it? :)

File: client/view/QueryLink/queryLink.js (hunk 1 above)
Comment (a-chabin) [reply client/view/QueryLink/queryLink.js:18]: Good idea. But it didn't work. :(

I'll check later why it didn't work for viewport (nothing happens either with the js method or when specifying the css property `scroll-behavior`)

**Pull request merged**